### DISCUSSION TITLE
CHANGELOG was removed

### DISCUSSION THREAD

**Discussion by krajek** [RehanSaeed/Serilog.Exceptions#166] [Q&A]
### Question

Hi @RehanSaeed ,

I saw that change log was removed by 40005d5ac036b27523599aa2afcb33955c613550.
Was it on purpose? Is there any other place to document changes for each version?

**Comment by RehanSaeed** [comment]
GitHub releases now kick of CI builds and automatically publish the NuGet package. You just need to bump the version number before you create a release.

https://github.com/RehanSaeed/Serilog.Exceptions/releases
